app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="HOME"/>
```

Comment:
Get used to set strings in Strings.xml. Just to get a good muscle memory :muscle: :stuck_out_tongue: